Scripts/Player/Entity.gd

```diff
@@ -11,28 +11,33 @@ enum {
 	ATTACK
 }
 
+
 onready var game_state = get_node("/root/GameState")
 onready var animation_player = $AnimationPlayer
 onready var animation_tree = $AnimationTree
+onready var swordHitbox = $Hitbox
 var animation_state = null
 
 func _ready():
 	game_state.connect("no_health", self, "queue_free")
 	if animation_tree:
 		 animation_state = animation_tree.get("parameters/playback")
-
+	
+	
 func _physics_process(delta):
 	pass
 
 func move_state(delta, movement_vector):
-	
+	var input_vector = Vector2.ZERO
```

Comment:
I guess that this is not needed in this PR?

Reply:
Yeah, that's outdated. Wonder how it got back in there haha.

Reply:
Yeah sorry I was trying something there, forgot to remove it